specification/loadtestservice/LoadTestService.Management/models.tsp

```diff
@@ -155,7 +155,17 @@ model LoadTestProperties {
 #suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "CheckQuotaAvailabilityResponse has a allOf relationship with Resource"
```

Comment:
same comment here
I think this suppress is not relevant anymore , wasn't either before too